canonical-kubernetes/addons/deis/steps/step-01_install-workflow

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+set -eux
+
+. "$CONJURE_UP_SPELLSDIR/sdk/common.sh"
+
+export KUBECONFIG=$HOME/.kube/config.$JUJU_MODEL
+
+WORK_DIR=`mktemp -d`
+
+cleanup() {
+    rm -rf "$WORK_DIR"
+}
+
+echo "Installing Deis CLI"
+curl -fsSL -o $HOME/bin/deis  https://storage.googleapis.com/workflow-cli-release/deis-stable-linux-amd64
+chmod +x $HOME/bin/deis
+
+echo "Installing Helm CLI"
+HELM_VERSION="v2.6.0"
+curl -fsSL -o $WORK_DIR/helm-stable.tar.gz https://storage.googleapis.com/kubernetes-helm/helm-"$HELM_VERSION"-linux-amd64.tar.gz
+tar -C $WORK_DIR -zxvf helm-stable.tar.gz
+mv $WORK_DIR/linux-amd64/helm $HOME/bin/helm
+
+init_count=0
+while ! helm init --upgrade; do
+    if [[ "$init_count" -gt 5 ]]; then
+        break
+    fi
+    ((init_count=init_count+1))
+    sleep 5
+done
+
+
+check_ready_replicas() {
+    local ready_replicas="$(kubectl -n kube-system get deployment tiller-deploy -o json|jq .status.readyReplicas)"
+    local replicas="$(kubectl -n kube-system get deployment tiller-deploy -o json|jq .status.replicas)"
+    [[ -z "$ready_replicas" ]] || ready_replicas=0
```

Comment:
jq can actually return `null` here which is considered an unbound variable. This only works because it doesn't actually fail and after 15 tries continues on happily. Everything I googled hasn't lead me to a decent solution to this. Suggestions welcomed.